app/models/pageflow/account_member_query.rb

```diff
@@ -0,0 +1,57 @@
+module Pageflow
+  class AccountMemberQuery < ApplicationQuery
+    class Scope < Scope
```

Comment:
This works but does not read well. Let's be explicit and write `ApplicationQuery::Scope`.